CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Release Versions:
 
 ## Upcoming changes (in development)
 
+- Remove timestamp setter and rename getter (#2)
```

Comment:
I made the mistake in the first PR - we normally order items in the changelog from top to bottom. Could you change it to the following?
```markdown
...
- Add pkg-config files for clproto (epfl-lasa#327)
- Use pkg-config to check for dependencies (epfl-lasa#328)
- Transfer repository ownership to AICA (#1)
- Remove timestamp setter and rename getter (#2)
```